src/material-experimental/mdc-chips/testing/chip-remove-harness.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {ChipRemoveHarnessFilters} from './chip-harness-filters';
+
+/** Harness for interacting with a mat-chip removal button in tests. */
+export class MatChipRemoveHarness extends ComponentHarness {
```

Comment:
The non-MDC harness has a `click` method so we should probably add it to this one too.

Reply:
Sorry, that's outside the scope of what I'd like to accomplish here.